src/scripts/main.js

```diff
@@ -22,7 +22,20 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  const list = document.createElement('ul');
+
+  for (const key in data) {
+    const listItem = document.createElement('li');
+
+    listItem.textContent = key;
+    list.append(listItem);
+
+    if (typeof data[key] === 'object') {
```

Comment:
the typeof empty object {} also 'object', that`s why you have such unnecessary elements
![image](https://user-images.githubusercontent.com/64437769/91033314-d91ff380-e60b-11ea-8a54-49baedb560d5.png)

Reply:
thanks, fixed it